packages/babel-plugin-transform-template-literals/src/index.js

```diff
@@ -75,41 +72,29 @@ export default declare((api, options) => {
           }
         }
 
-        // Generate a unique name based on the string literals so we dedupe
-        // identical strings used in the program.
-        const rawParts = raws.map(s => s.value).join(",");
-        const name = `${helperName}_${raws.length}_${rawParts}`;
-
-        let templateObject = this.templates.get(name);
-        if (templateObject) {
-          templateObject = t.cloneNode(templateObject);
-        } else {
-          const programPath = path.find(p => p.isProgram());
-          templateObject = programPath.scope.generateUidIdentifier(
-            "templateObject",
-          );
-          this.templates.set(name, templateObject);
-
-          const helperId = this.addHelper(helperName);
-          const callExpressionInput = [];
-          callExpressionInput.push(t.arrayExpression(strings));
-
-          if (!isStringsRawEqual) {
-            callExpressionInput.push(t.arrayExpression(raws));
-          }
+        const programPath = path.find(p => p.isProgram());
```

Comment:
You can use `path.scope.getProgramParent()`

Reply:
Done.